spring-cloud-generator/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>spring-cloud-gcp</artifactId>
+        <groupId>com.google.cloud</groupId>
+        <version>4.1.5-SNAPSHOT</version><!-- {x-version-update:spring-cloud-gcp:current} -->
+    </parent>
+
+    <groupId>com.google.api</groupId>
+    <artifactId>spring-cloud-generator</artifactId>
```

Comment:
Is there a Maven release exclusion for this module? I don't think we want to be released together with other Spring Cloud GCP maven modules just yet.

Reply:
Addressing this comment along with https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1686#discussion_r1169187271:

This PR adds `spring-cloud-generator` as a module under spring-cloud-gcp's default profile:

https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/7d14fa45496bbe71ac252f8fd98ddb9489aec463/pom.xml#L342-L349

But not the release profile:
https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/7d14fa45496bbe71ac252f8fd98ddb9489aec463/pom.xml#L565-L570

This is similar to what we did for the `spring-cloud-previews` module before their actual release as part of https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1415. 

Please let me know if there is anything I've overlooked though - I can also double check with running a stage-only job from this PR before merging.

Reply:
The `spring-cloud-previews` also has the following section:
```
  <build>
    <plugins>
      <plugin>
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-deploy-plugin</artifactId>
        <version>${maven-deploy-plugin.version}</version>
        <configuration>
          <skip>true</skip>
        </configuration>
      </plugin>
    </plugins>
  </build>
```

I think we put it in as a second layer prevention in addition to the profiles in parent pom.

Reply:
Thanks for the pointer! Added this skip configuration to the pom in [079b08f](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1686/commits/079b08fd47bca2b36db0757a160a6c44f2d6c0c5).